docs/concepts/00_what_is_hl_aries.md

```diff
@@ -2,7 +2,7 @@
 
 Hyperledger Aries provides a shared, reusable, interoperable tool kit designed for initiatives and solutions focused on creating, transmitting and storing verifiable digital credentials. It is infrastructure for blockchain-rooted, peer-to-peer interactions. This project consumes the cryptographic support provided by Hyperledger Ursa, to provide secure secret management and decentralized key management functionality. _(1)_
 
-## Where did Aries come from?
+## Motivation
```

Comment:
I think the text in this section will be confusing as "motivation".

Reply:
Should it be reverted @troyronda